src/renderer/renderer.cpp

```diff
@@ -187,11 +187,14 @@ void BaseRenderer::DrawProcessedText_BasicFont(ProcessedTextGlyph* text,
           DrawSprite(glyph, dest, color);
           dest.X++;
           dest.Y++;
+          break;
```

Comment:
This fall through is intentional

Reply:
Apparently there's no standard way to annotate explicit fallthrough pre-C++17. I've added an __attribute__ that should handle this for GCC and clang, silencing on MSVC will have to wait until we've upgraded the language version.

Reply:
Unfortunately it doesn't seem to work at all on MSVC, so probably best to just get rid of the fallthrough altogether

Reply:
Done.